spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/ReadConf.scala

```diff
@@ -20,15 +21,18 @@ object ReadConf {
   val DefaultSplitSize = 100000
   val DefaultFetchSize = 1000
   val DefaultConsistencyLevel = ConsistencyLevel.LOCAL_ONE
+  import CassandraConnectorConf.convert
+  def fromSparkConf(conf: SparkConf): ReadConf = getReadConf(conf)
 
-  def fromSparkConf(conf: SparkConf): ReadConf = {
+  def fromSparkConf(conf: SparkConf, cluster: Option[String] = None): ReadConf = getReadConf(conf, cluster)
+
+  private def getReadConf(conf: SparkConf, cluster: Option[String] = None): ReadConf = {
     ReadConf(
-      fetchSize = conf.getInt("spark.cassandra.input.page.row.size", DefaultFetchSize),
-      splitSize = conf.getInt("spark.cassandra.input.split.size", DefaultSplitSize),
+      fetchSize = conf.getInt(convert("spark.cassandra.input.page.row.size", cluster), DefaultFetchSize),
```

Comment:
@jacek-lewandowski  All I need is to get cluster level readconf from single sparkconf. If you don't think it's related to cluster, can you provide a solution here, I'd like to take it. I don't really care about the changes will be in ReadConf classes or somewhere else.